swagger.yaml

```diff
@@ -124,6 +124,136 @@ paths:
           description: "Internal Server Error"
           schema:
             $ref: "#/definitions/500Response"
+  /user:
+    post:
+      tags:
+        - User
+      summary: Register user into system
+      parameters:
+        - name: user
+          in: body
+          description: Create user
+          schema:
+            $ref: "#/definitions/user"
```

Comment:
We should define definitions for Create or Update body separately it should be not just `User` which we will use in responses.